src/src/AdminBundle/Repository/AttendeeRepository.php

```diff
@@ -86,11 +91,11 @@ public function getCountOfAttendeesFilteredWithoutEvent(
     private function getSearchByEventQuery($selectQuery, $nameEmail, $event, $isSubscribed)
     {
         $result = $selectQuery
-            ->innerJoin('AdminBundle:Attendee', 'a', 'WITH', 'a.id = r.attendeeId')
+            ->innerJoin(Attendee::class, 'a', 'WITH', 'a.id = r.attendee')
             ->where('a.email LIKE :name OR a.firstName LIKE :name OR a.lastName LIKE :name')
```

Comment:
`like` hlada substringy vsak?

Reply:
odkial mame `r`?

Reply:
nestaci pouzit `where` namiesto `andwhere`?
`r.attendee` nema byt nahodou `r.attendee_id`? Lebo tak je to v navrhu.
Preco pouzivame `r.events` pod eventS je len jeden event? Navyse, nema to byt event_id ked uz? Ten navrh sa celkom odlisuje od tohoto... ale pamatam si, ze tam bol zmatok medzi nazvami typu `xxx` a `xxx_id`.

Reply:
Logiku som nemenil, len som fixol to co bolo rozbite. Toto kodila @viktorria, ona ti bude moct zodpovedat na vsetky tie otazky. Nieje to vsak sucastou tohoto PR.

Reply:
Cize to nemam riesit v tomto pr a dat approve?

Reply:
jop.

Reply:
Oki